articles/users/search.md

```diff
@@ -94,11 +113,72 @@ This endpoint is **eventually consistent**, and as such, we recommend that you u
 
 We do **not** recommend that you use this endpoint for:
 
-* Operations that require immediate consistency - please use the [Users by Email endpoint](#users-by-email) for such actions
+* Operations that require immediate consistency - please use the [Users by Email endpoint](#users-by-email) or the [Users by ID endpoint](#users-by-id) for such actions
 * User exports - please use the [User Export endpoint](#user-export) for such actions
-* Operations that require user search as part of the Authentication Pipeline - please use the [Users by Email endpoint](#users-by-email) for such actions
+* Operations that require user search as part of the Authentication Pipeline - please use the [Users by Email endpoint](#users-by-email) or the [Users by ID endpoint](#users-by-id) for such actions
 * Searching for Users for [Account Linking](/link-accounts) by Email - please use the [Users by Email endpoint](#users-by-email) for such actions
 
+## Users by ID
+
+The [`GET /api/v2/users/{id}` endpoint](/api/management/v2#!/Users/get_users_by_id) allows you to retrieve a specific user using their Auth0 user ID.
+
+*Required Scopes*: `read:users`
+
+```har
+{
+	"method": "GET",
+	"url": "https://${account.namespace}/api/v2/users/USER_ID",
+	"headers": [{
+		"name": "Authorization",
+		"value": "Bearer YOUR_MGMT_API_ACCESS_TOKEN"
+	}]
+}
+```
+
+**Sample Response**
+
+Successful calls to the endpoint returns a JSON object similar to the following:
+
+```json
+{
+  "email": "john.doe@gmail.com",
+  "email_verified": false,
+  "username": "johndoe",
+  "phone_number": "+199999999999999",
+  "phone_verified": false,
+  "user_id": "usr_5457edea1b8f33391a000004",
+  "created_at": "",
+  "updated_at": "",
+  "identities": [
+    {
+      "connection": "Initial-Connection",
+      "user_id": "5457edea1b8f22891a000004",
+      "provider": "auth0",
+      "isSocial": false
+    }
+  ],
+  "app_metadata": {},
+  "user_metadata": {},
+  "picture": "",
+  "name": "",
+  "nickname": "",
+  "multifactor": [
+    ""
+  ],
+  "last_ip": "",
+  "last_login": "",
+  "logins_count": 0,
+  "blocked": false,
+  "given_name": "",
+  "family_name": ""
+}
+```
+
+This endpoint is **immediately consistent**, and as such, we recommend that you use this endpoint for:
+
+* User searches run during the authentication/authorization process 
```

Comment:
Here you use `authentication/authorization process`. It's clearer than `authentication pipeline`. I also think you could lose the `/authorization` and simplify more.